webextensions/api/menus.json

```diff
@@ -38,7 +38,7 @@
             "support": {
               "chrome": {
                 "alternative_name": "contextMenus.ACTION_MENU_TOP_LEVEL_LIMIT",
-                "version_added": "≤53"
+                "version_added": "39"
```

Comment:
:x: Based on https://github.com/chromium/chromium/commit/69e1c12d5b212fc29af5f31982ccca8cb66e5c01, this landed in [38.0.2123.0](https://github.com/chromium/chromium/releases/tag/38.0.2123.0), before Chrome 38[.0.2125] landed, so it should be in Chrome 38.